src/Components/Components/src/Rendering/HtmlRenderer.cs

```diff
@@ -28,14 +29,15 @@ public class HtmlRenderer : Renderer
         /// </summary>
         /// <param name="serviceProvider">The <see cref="IServiceProvider"/> to use to instantiate components.</param>
         /// <param name="loggerFactory">The <see cref="ILoggerFactory"/>.</param>
-        /// <param name="dispatcher">The <see cref="IDispatcher"/> to be for invoking user actions into the <see cref="Renderer"/> context.</param>
         /// <param name="htmlEncoder">A <see cref="Func{T, TResult}"/> that will HTML encode the given string.</param>
-        public HtmlRenderer(IServiceProvider serviceProvider, ILoggerFactory loggerFactory, IDispatcher dispatcher, Func<string, string> htmlEncoder)
-            : base(serviceProvider, loggerFactory, dispatcher)
+        public HtmlRenderer(IServiceProvider serviceProvider, ILoggerFactory loggerFactory, Func<string, string> htmlEncoder)
+            : base(serviceProvider, loggerFactory)
         {
             _htmlEncoder = htmlEncoder;
         }
 
+        public override Dispatcher Dispatcher { get; } = new RendererSynchronizationContextDispatcher();
```

Comment:
I'm not completely certain why this one explicitly constructs a `RendererSynchronizationContextDispatcher` instead of using `Renderer.CreateDefault`, but I guess the result will be the same.

Reply:
Will review these, I think this was done before I had to make `CreateDefault`.